apps/AppMakefileMbed.mk.posix

```diff
@@ -0,0 +1,93 @@
+################################################################################
+## Linux flavored Makefile for getting Mbed code onto microcontrollers
+################################################################################
```

Comment:
Is there any reason to keep these rules separate from `AppMakefileMbed.mk`? We don't plan on having windows support anytime soon, and although the top of the file claims its for linux, it does work on macOS too.